jetty-quickstart/src/main/config/etc/jetty-quickstart.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <New id="props" class="java.util.HashMap">
+    <Call name="put">
+       <Arg>jetty.quickstart.mode</Arg>
+       <Arg><Property name="jetty.quickstart.mode"/></Arg>
+    </Call>
+    <Call name="put">
+       <Arg>jetty.quickstart.generateOrigin</Arg>
+       <Arg><Property name="jetty.quickstart.generateOrigin"/></Arg>
+    </Call>
+    <Call name="put">
+       <Arg>jetty.quickstart.origin</Arg>
+       <Arg><Property name="jetty.quickstart.origin"/></Arg>
```

Comment:
Can we just combine these now to a single parameter, that if non null will be used as the attribute name to which the origin is generated. If not set, then origin is not generated